### PR TITLE
chore: upgrade tekton chart to 0.1.0

### PR DESCRIPTION
**Pull request by ankitm123**
Signed-off-by: ankitm123 <ankitmohapatra123@gmail.com>

This upgrades tekton to 0.32.4

### CODE REVIEW

**Comment by ankitm123** [comment]
Needs to be tested on k3s by @rajatgupta24 as well.

**Comment by ankitm123** [comment]
@rajatgupta24 tested it locally on his cluster, and it passed basic checks
- pipeline was triggered
- logs and status of the pipeline was updated in the UI

**Comment by ankitm123** [comment]
Slack thread: https://kubernetes.slack.com/archives/C9MBGQJRH/p1663931082013609

File: charts/cdf/tekton-pipeline/defaults.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 gitUrl: https://github.com/cdfoundation/tekton-helm-chart
 namespace: tekton-pipelines
-version: 0.29.0
+version: 0.1.0
```
Comment (nicolaerosia) [inline charts/cdf/tekton-pipeline/defaults.yaml:3]: i think this should be 0.2.0, https://github.com/cdfoundation/tekton-helm-chart/blob/main/charts/tekton-pipeline/Chart.yaml#L4

Comment (ankitm123) [reply charts/cdf/tekton-pipeline/defaults.yaml:3]: So 0.2.0 has one extra change over 0.1.0. I wanted to keep the change exclusive to tekton version upgrade.
Next week we will do 0.2.0

**Pull request merged**